### PR TITLE
Add release notes for 0.4.15

### PR DESCRIPTION
**Pull request by jni**
# Description

These are the autogenerated (manually rearranged) notes for 0.4.15, for which we release the rc tomorrow. Nothing particularly controversial in this release, mostly fixes and documentation improvements, so hopefully a smooth release process!

### CODE REVIEW

File: docs/release/release_0_4_15.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- Don't use git+https to install on CI (#4116)
+- Add markupsafe pin to bundle build (#4118)
+- Don't pip install from github in comprehensive tests (avoid another LFS leak) (#4141)
+- fix make docs for non editable installs (#4156)
+
+## Other Pull Requests
+
+- [pre-commit.ci] pre-commit autoupdate (#4032)
+- remove leftover comments on `Points.n_dimensional` (#4052)
+- Drop python 3.7 (#4063)
```
Comment (tlambert03) [inline docs/release/release_0_4_15.md:110]: perhaps note at the top

File: docs/release/release_0_4_15.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## New Features
+
+- Add edge_width per-point and edge_width_is_relative (#3999)
+- Support error reporting via napari-error-reporter (#4055)
+
+## Improvements
+
+- avoid repeated computation of layers.extent when adding a new Labels layer (#4037)
+- Add style encoding for strings (#4051)
```
Comment (tlambert03) [inline docs/release/release_0_4_15.md:47]: highlight?  not sure how big of a jump this was, but if there's any new feature APIs to use, they could be mentioned.

**Review by tlambert03 — APPROVED** [review]
looks about right to me!

File: docs/release/release_0_4_15.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- New documentation infrastructure (#4047)
+- Add Grzegorz to 0.4.14 release notes (#4061)
+- remove `--checkout npe2` instructions for cookiecutter in documentation (#4092)
+- Clarify that napari should be installed (#4101)
+- Move workshop out of tutorials into usage (#4123)
+- Refactor layers content (#4124)
+- Organize tutorials sections (#4137)
+- Add release management docs (#4139)
+- Use napari-sphinx-theme in docs build (#4147)
+- Add new core developers (#4157)
```
Comment (alisterburt) [inline docs/release/release_0_4_15.md:73]: Potential highlight? 🙂

File: docs/release/release_0_4_15.md (hunk 2 above)
Comment (alisterburt) [reply docs/release/release_0_4_15.md:47]: +1 - especially because the doc at the top of the PR shows where this is taking us :)

Comment (jni) [reply docs/release/release_0_4_15.md:47]: I agree with the sentiment, but I actually hesitate to highlight until we have a new API usage example corresponding to it. I'd like @andy-sweet to chime in as to how much he thinks we should publicise in the new release. We can easily move this around after rc0 if we decide to highlight it more. :+1:

File: docs/release/release_0_4_15.md (hunk 3 above)
Comment (jni) [reply docs/release/release_0_4_15.md:73]: Well, both were added before 0.4.15, so it's a bit weird... LOL We just forgot to update the docs

**Comment by jni** [comment]
@alisterburt are you able to give this and my comments above a final look through before I merge and make the rc? :tada: As a reminder the release notes are not final, and will be updated over the coming week.

Actually I'll update the title to rc0 so things aren't confusing on the website... :joy:

**Review by alisterburt — APPROVED** [review]
Looks good to me! :)

File: docs/release/release_0_4_15.md (hunk 3 above)
Comment (alisterburt) [reply docs/release/release_0_4_15.md:73]: okay by me either way here :)

File: docs/release/release_0_4_15.md (hunk 2 above)
Comment (alisterburt) [reply docs/release/release_0_4_15.md:47]: that's fair! We can wait on Andy and update as necessary

File: docs/release/release_0_4_15.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+drops support for Python 3.7, which enables us to simplify many code paths.
+From 0.4.15, napari supports Python 3.8, 3.9, and 3.10.
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+This release is focused on documentation improvements and bug fixes, with few
+change's to napari's API. The napari documentation is now entirely contained,
```
Comment (alisterburt) [inline docs/release/release_0_4_15.md]:
```suggestion
changes to napari's API. The napari documentation is now entirely contained,
```

**Comment by jni** [comment]
Ha, thanks for the quick fix! :joy:

File: docs/release/release_0_4_15.md (hunk 2 above)
Comment (andy-sweet) [reply docs/release/release_0_4_15.md:47]: Like @jni said, I think we should avoid highlighting it in this release, as it's not publicly exposed, there's no examples or documentation (other than the docstrings), and it's not being used internally by napari yet.

**Pull request merged**